docs/telemetry.rst

```diff
@@ -31,6 +31,8 @@ You probably want to gain additional insights from a race. Therefore, we have ad
    shard-stats                 Shard Stats                 Regularly samples nodes stats at shard level
    data-stream-stats           Data Streams Stats          Regularly samples data streams stats
 
+.. warning::
```

Comment:
This should be a separate warning block, maybe before the output of the command line example. The sentence below ("Keep in mind that each telemetry device may incur a runtime overhead which can skew results.") is actually part of the command line output of `esrally list telemetry` and should not be removed.

Reply:
The alternative I see is duplicating the warning, please tell me if that's better

Reply:
Yes, I'd prefer a separate warning (above the example) directly in the docs. This fact is important enough that it warrants a warning in the docs and in the command line output IMHO.

Reply:
Sorry, I meant "I've updated the docs like you said, please take a look". I wasn't trying to ask you to repeat your argument a second time, hoping that would change your mind :D